src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -107,15 +107,19 @@ const UsersFragment = ({
   }, []);
 
   React.useEffect(() => {
+    if (searchText?.length === 0) {
+
     handleActiveSections();
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition `if (searchText?.length === 0)` seems redundant because the same check is performed inside the `handleActiveSections` function. You can remove this condition and directly call `handleActiveSections`.
```diff
-    if (searchText?.length === 0) {
-      handleActiveSections();
-    }
+    handleActiveSections();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->